supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesParameterConversions.scala

```diff
@@ -26,7 +27,25 @@ trait PipelinesParameterConversions {
         Key.Tag -> Value.Localization,
         Key.InputName -> fileInput.name
       )
-      cloudSdkBashAction(localizeFile(fileInput.cloudPath, fileInput.containerPath))(mounts, labels = labels)
+
+      if (fileInput.cloudPath.pathAsString.startsWith("dos://")) {
+        import cromwell.backend.google.pipelines.v2alpha1.api.ActionCommands.ShellPath
+        val config = ConfigFactory.load
+        val demoDosDockerImage = config.getString("demo.dos.docker-image")
+        val demoDosCommandTemplate = config.getString("demo.dos.command-template")
```

Comment:
Would you consider making these abstract strings on the trait and filled in elsewhere?  It'd be nice to not add static(-ish, I realize it's in a def) config loads, and if inevitable, make the caller do it and provide these values.

Reply:
Sure, will go over a plan tomorrow. Happy to implement whatever for the demo.

I changed course and tried to keep this PR as isolated as technically possible. I'm keeping one eye on what will be needed for `read_lines` support etc post demo. I foresee possibly (probably?) refactoring a bit of this code to something more permanent... something... if only... 🤔

Will likely pull in @Horneth for his expertise too.

Reply:
Created a ticket for following up on this comment: https://github.com/broadinstitute/cromwell/issues/3858

Merging as is for now.